example/20-crd-bastion.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Can you also add an example file for a `Bastion` resource?

Reply:
add example  https://github.com/gardener/gardener-extension-provider-gcp/pull/288/commits/33570f2898da5b814bb3aebcb5c8766c0f4be77a